.github/ISSUE_TEMPLATE/question.md

```diff
@@ -9,8 +9,10 @@ assignees: ''
 
 Instructions
 ============
-Here, you can ask a question about this STL implementation,
-and a maintainer will answer.
+Here, you can ask a question about this STL implementation, and a maintainer
+will answer. Please read the examples below, then delete all of this text and
+replace it with your question. If you aren't sure whether a question is
+on-topic, just go ahead and ask! :-)

```

Comment:
Probably not an issue but there's perhaps a slight ambiguity as to whether this means ask if the question is on topic or ask the question they were going to ask anyway.

Reply:
Good catch, fixed!